ompi/mca/coll/hcoll/coll_hcoll_module.c

```diff
@@ -206,7 +206,7 @@ static int hcoll_comm_attr_del_fn(MPI_Comm comm, int keyval, void *attr_val, voi
     mca_coll_hcoll_module_t *hcoll_module;
     hcoll_module = (mca_coll_hcoll_module_t*) attr_val;
 
-#if HCOLL_API >= HCOLL_VERSION(3,7)
+#ifdef HAVE_HCOLL_CONTEXT_FREE
```

Comment:
Another minor style nit: if you're going to `#if !defined(...` in one place, you should probably make the style consistent in the rest of the file (e.g., `#if defined(...` vs. `#ifdef`).